src/python/pants/bin/local_pants_runner.py

```diff
@@ -154,16 +188,37 @@ def __init__(self, build_root, exiter, options, options_bootstrapper, build_conf
     self._profile_path = profile_path
 
     self._run_start_time = None
+    self._run_tracker = None
+    self._reporting = None
+    self._repro = None
     self._global_options = options.for_global_scope()
 
   def set_start_time(self, start_time):
```

Comment:
It's odd for this all to be inside of `def set_start_time`.

Reply:
Sure, but I was mostly following the comment immediately below:
> `# Launch RunTracker as early as possible (before .run() is called).`

Modifying `set_start_time()` seemed like exactly the right place, as it necessarily comes before `run()`. If that introduces other issues, it can be changed.